.circleci/config.yml

```diff
@@ -592,7 +592,7 @@ jobs:
 
   test-discovery-provider:
     docker:
-      - image: cimg/python:3.9-node
+      - image: cimg/python:3.7-node
```

Comment:
this has the negative impact of downgrading our python version right? it might make sense to keep this one as 3.9 for now (since I think that might cause some pain)

Reply:
https://app.circleci.com/pipelines/github/AudiusProject/audius-protocol/26304/workflows/30546281-7a03-442c-afd7-d2e9a4ae6fc8/jobs/314398